src/transformState.js

```diff
@@ -5,7 +5,26 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  
+  for (let i = 0; i < actions.length; i++) {
+    if (actions[i].type === 'addProperties') {
+      for (const prop in actions[i].extraData) {
+        state[prop] = actions[i].extraData[prop];
+      }
+    }
+
+    if (actions[i].type === 'removeProperties') {
+      for (const word in actions[i].keysToRemove) {
```

Comment:
what's a word? that's a key of object or a property, so name your variables as they should be named

Reply:
Also `keysToRemove` is an array, you can use `for ... of` here